src/app/shared/components/wrapper/wrapper.component.spec.ts

```diff
@@ -1,14 +1,20 @@
 import { ComponentFixture, TestBed } from '@angular/core/testing';
 
 import { WrapperComponent } from './wrapper.component';
+import { RouterTestingModule } from '@angular/router/testing';
+import { CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
 
 describe('WrapperComponent', () => {
   let component: WrapperComponent;
   let fixture: ComponentFixture<WrapperComponent>;
 
   beforeEach(async () => {
     await TestBed.configureTestingModule({
-      declarations: [ WrapperComponent ]
+      imports: [RouterTestingModule],
+      declarations: [
+        WrapperComponent,
+      ],
+      schemas: [CUSTOM_ELEMENTS_SCHEMA]
```

Comment:
It is better to declare needed components and avoid CUSTOM_ELEMENTS_SCHEMA. Because it is more likely workaround, but not solution. On our project we defined stub components and add this stub components to declaration.

Reply:
fixed